### ISSUE TITLE
Is it appropriate to perform functional enrichment on up vs down regulated genes based on fold change? 

### ISSUE THREAD

**Issue by AHuffmyer** [AHuffmyer/EarlyLifeHistory_Energetics#5]
Discussion in Roberts Lab that it isn't good practice to split up vs down regulated genes based on FC. Maybe instead we need to do functional enrichment on all DEGS regardless of direction and then plot expression to give more context.

**Comment by hputnam** [comment]
Whats the rationale? Aren't all comparisons just hypotheses?

**Comment by AHuffmyer** [comment]
I agree - and I have seen its common in the literature to split up vs down regulated genes and do separate enrichment analyses. [This paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3899863/) shows higher detection when done separately rather than together.

**Comment by AHuffmyer** [comment]
Argument in the discussion is that you can't really say up vs down regulated based on Fold Change, because it doesn't give the whole picture of transcription activity. But you could say that about any thing we measure that is a snap shot in time.
